app/components/emoji_keyboard.js

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import {
+    View,
+    Text,
+    StyleSheet,
+    FlatList,
+    TouchableOpacity
+} from 'react-native';
+import {KeyboardRegistry} from 'react-native-keyboard-input';
+import {store} from 'app/mattermost';
+import {CategoryNames, EmojiIndicesByCategory, Emojis} from 'app/utils/emojis';
+import Emoji from 'app/components/emoji/index';
+import PropTypes from 'prop-types';
+
+function buildCategory(category) {
```

Comment:
I'm not sure, but I think I'll use a lambda for this until we decide where to put it. The data being built is very `FlatList`-centric and could be a little bit redundant to have it in emoji utils since it already has that info but in different shape.